resources/assets/js/components/Application/ExperienceModals/PersonalExperienceModal.tsx

```diff
@@ -0,0 +1,381 @@
+
+export const PersonalExperienceModal: React.FC<PersonalExperienceModalProps> = ({
+  modalId,
+  experiencePersonal,
+  jobId,
+  requiredSkills,
+  savedRequiredSkills,
+  optionalSkills,
+  savedOptionalSkills,
+  experienceRequirments,
+  useAsEducationRequirement,
+  parentElement,
+  visible,
+  onModalCancel,
+  onModalConfirm,
+}) => {
+  const intl = useIntl();
+  const locale = getLocale(intl.locale);
+
+  const originalExperience = experiencePersonal ?? newPersonalExperience();
+
+  const skillToName = (skill: Skill): string =>
+    localizeFieldNonNull(locale, skill, "name");
+
+  const initialFormValues = dataToFormValues(
+    {
+      experiencePersonal: originalExperience,
+      savedRequiredSkills,
+      savedOptionalSkills,
+      useAsEducationRequirement,
+    },
+    locale,
+  );
+
+  const validationSchema = Yup.object().shape({
+    ...skillValidationShape,
+    ...educationValidationShape,
+    ...validationShape(intl),
+  });
+
+  const detailsSubform = (
+    <div data-c-container="medium">
+      <div data-c-grid="gutter(all, 1) middle">
+        <FastField
+          id="title"
+          name="title"
+          type="text"
+          grid="base(1of1)"
+          component={TextInput}
+          required
+          label={intl.formatMessage(messages.titleLabel)}
+          placeholder={intl.formatMessage(messages.titlePlaceholder)}
+        />
+        <FastField
+          id="description"
+          type="text"
+          name="description"
+          component={TextAreaInput}
+          required
+          grid="tl(1of1)"
+          label={intl.formatMessage(messages.descriptionLabel)}
+          placeholder={intl.formatMessage(messages.descriptionPlaceholder)}
+          wordLimit={DESCRIPTION_WORD_LIMIT}
+        />
+        <div data-c-input="checkbox(group)" data-c-grid-item="base(1of1)">
+          <label>{intl.formatMessage(messages.isShareableLabel)}</label>
+          <FastField
+            id="isShareable"
+            name="isShareable"
+            component={CheckboxInput}
+            grid="base(1of1)"
+            label={intl.formatMessage(messages.isShareableInlineLabel)}
+          />
+        </div>
+        <FastField
+          id="startDate"
+          name="startDate"
+          component={DateInput}
+          required
+          grid="base(1of1)"
+          label={intl.formatMessage(messages.startDateLabel)}
+          placeholder={intl.formatMessage(messages.datePlaceholder)}
+        />
+        <Field
+          id="isActive"
+          name="isActive"
+          component={CheckboxInput}
+          grid="tl(1of2)"
+          label={intl.formatMessage(messages.isActiveLabel)}
+        />
+        <Field
+          id="endDate"
+          name="endDate"
+          component={DateInput}
+          grid="base(1of2)"
+          label={intl.formatMessage(messages.endDateLabel)}
+          placeholder={intl.formatMessage(messages.datePlaceholder)}
+        />
```

Comment:
The end date has to be not required if they check the "Activity ongoing" box. This bit in the validation covers this situation:
```
endDate: Yup.date().when("isActive", {
      is: false,
      then: Yup.date()
        .required(conditionalRequiredMsg)
        .min(Yup.ref("startDate"), afterStartDateMsg),
      otherwise: Yup.date().min(Yup.ref("startDate"), afterStartDateMsg),
    }),```